samples/csharp/getting-started/DatabaseIntegration/DatabaseIntegration/Program.cs

```diff
@@ -0,0 +1,122 @@
+
+        /// <summary>
+        /// Populates the database with the AdultCensus dataset.
+        /// </summary>
+        public static void Seeding()
+        {
+            string adultDataset = DownloadAdultDataset();
+            using (var db = new AdultCensusContext())
+            {
+                // Ensure that we have a clean database to start with.
+                db.Database.EnsureDeleted();
+                db.Database.EnsureCreated();
+
+                // Parse the downloaded file
+                var data = File.ReadLines(adultDataset)
+                    .Skip(1) // Skip the header row
+                    .Select(l => l.Split(','))
+                    .Where(row => row.Length > 1)
+                    .Select(row => new AdultCensus()
+                    {
+                            Age = int.Parse(row[0]),
+                            Workclass = row[1],
+                            Education = row[3],
+                            MaritalStatus = row[5],
+                            Occupation = row[6],
+                            Relationship = row[7],
+                            Race = row[8],
+                            Sex = row[9],
+                            CapitalGain = row[10],
+                            CapitalLoss = row[11],
+                            HoursPerWeek = int.Parse(row[12]),
+                            NativeCountry = row[13],
+                            Label = (int.Parse(row[14]) == 1) ? true : false
+                    });
+
+                // Add the data into the database
+                db.AdultCensus.AddRange(data);
+
+                var count = db.SaveChanges();
+                Console.WriteLine($"Save results {count}");
+            }
+        }
+
+        public static void Main()
+        {
+            // Seed the database with the dataset.
+            Seeding();
+
+            using(var db = new AdultCensusContext())
+            {
+                // Set the query tracking behavior to be NoTracking, this is particularly useful because
+                // our scenarios our read-only with the data. Not having tracking will  
+                db.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
+                var mlContext =  new MLContext();
+               
+                // Query our training data from the database. This query is selecting everything from the AdultCensus table. The
+                // result is then loaded by ML.Net through the LoadFromEnumerable. LoadFromEnumerable returns an IDataView which
+                // can be consumed by an ML.Net pipeline.
+                // NOTE: For training, ML.Net requires that the training data is processed in the same order to produce consistent results.
+                // Therefore we are sorting the data by the AdultCensusId, which is an auto-generated id.
+                var dataView = mlContext.Data.LoadFromEnumerable(db.AdultCensus.OrderBy(x=>x.AdultCensusId).ToList<AdultCensus>());
```

Comment:
`ToList<AdultCensus>()` will bring all the data into memory, defeating everything else we are doign to avoid that. Just pass the query (which is already an `IEnumerable<AdultCensus>()`) to the next API.